Dockerfile

```diff
@@ -67,6 +70,23 @@ RUN wget https://github.com/antham/ghokin/releases/download/v1.6.1/ghokin_linux_
 # Set python to be python3
 RUN alternatives --set python /usr/bin/python3
 
+# Because lcov is not available in centos8 repos or eple-release, we install from source
+RUN git clone https://github.com/linux-test-project/lcov.git && cd lcov && \
+    git checkout v1.15 && \
+    make dist && \
+    yum install lcov-1.15-1.noarch.rpm -y && \
+    make check && \
+    cd .. && \
+    rm lcov -rf
+ 
+# We use this to convert lcov format coverage data to cobertura
+RUN pip3 install lcov_cobertura 
```

Comment:
Can this not be in the above list of pip3 installs? We have test stuff there already

Reply:
Fixed.